includes/class-wp-security-bp-json.php

```diff
@@ -89,6 +101,7 @@ private function validate( $args = array() ) {
 	public function pass( $args = array() ) {
 
 		$this->validate( $args );
```

Comment:
@jbaldrich 
Is `$this->json['status'] = passed` deprecated or we just missed it?

Reply:
Ok. I've just seen we only check for fail status at frontend. It's a good practice?

Reply:
Not at all